docs/content/development/extensions-core/namespaced-lookup.md

```diff
@@ -13,14 +13,15 @@ Make sure to [include](../../operations/including-extensions.html) `druid-namesp
 ## Configuration
```

Comment:
can we drop a phrase or 2 about what is cached means ? like `CachedLookup provides a global pool of memory to cache lookups ...`